.github/workflows/test-and-publish-nightly.yml

```diff
@@ -33,28 +33,36 @@ jobs:
   build-binaries-and-test:
     strategy:
       matrix:
+        runson:
+          - {
+              display: linux-x64,
+              name: "codebuild-async-profiler-nightly-builds-x86-${{ github.run_id }}-${{ github.run_attempt }}"
+            }
+        java-version: [11, 17, 21, 23]
```

Comment:
Why was it restructured this way? (`linux-x64` moved out of `include`; braces around display/name)

Reply:
This produces a cartesian product of `java-version X linux-x64`, then includes two single items `(macoos-14-arm64, 11)` and `(linux-arm64, 11)`.

Reply:
And what about curly braces?

Reply:
When curlies are removed, GHA does not parse it correctly, open to suggestions.